.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Launch ACAS and run python tests
+
+on:
+  push:
+    branches: ["main", "release/*"]
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  acas:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout acasclient
+        uses: actions/checkout@v3
+        with:
+          path: acasclient
+      - name: Set Up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      - name: Install acasclient and dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ./acasclient
+      # The following steps are all to figure out the right ACAS_REF (git ref) and ACAS_TAG (docker tag) of ACAS to use
```

Comment:
Please just review only this `run_tests.yml` file, especially this section where we figure out `ACAS_REF` and `ACAS_TAG`